IPython/core/interactiveshell.py

```diff
@@ -252,13 +252,17 @@ class ExecutionInfo:
     Stores information about what is going to happen.
     """
     raw_cell = None
+    transformed_cell = None
     store_history = False
     silent = False
     shell_futures = True
     cell_id = None
 
-    def __init__(self, raw_cell, store_history, silent, shell_futures, cell_id):
+    def __init__(
+        self, raw_cell, transformed_cell, store_history, silent, shell_futures, cell_id
```

Comment:
For backward compatibility:

```suggestion
        self, raw_cell, store_history, silent, shell_futures, cell_id, transformed_cell=None
```